src/components/message/demo/warning-message.twig

```diff
@@ -0,0 +1,13 @@
+{% embed '../message.twig' with {
+  includeClose: true,
+  isWarning: true,
+  class: 'c-message--warning' } 
+%}
+  <p id="MessageInfo">
+    {% block content %}
+      You seem to be offline at the moment.
+      <span class="u-hidden u-md-inlineBlock">We&rsquo;ll keep trying to
```

Comment:
- These utilities don't exist yet. Can we remove this content?
- Can we replace `&rsquo;` with a unicode `’`?